main.go

```diff
@@ -173,7 +175,9 @@ func main() {
 			log.Fatalln(err)
 		}
 
-		fmt.Println(arg)
+		if !quiet {
+			fmt.Println(arg)
+		}
```

Comment:
If there is an error, would it possible to still print the command? It makes it easier to debug generation issues.

Reply:
Ya, that makes sense to me.  The error would be any error that occurs after this line, right?

I could handle it by adding something like this:

```go
} else {
    log.SetPrefix(fmt.Sprintf("command %s", arg))
}
```
but that might make for some strange log messages.


Another option would be moving everything below this out into a new function which returns errors instead of `log.Fatal()` and adding the extra text here in the `if err != nil`.

WDYT?

Reply:
Looking at it again, I guess it's really only the `protoc.run()` error that needs to be handled, so I added a `log.Println(err)` to that if block.